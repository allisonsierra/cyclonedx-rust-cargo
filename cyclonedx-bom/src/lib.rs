pub mod errors;
pub mod models;

pub mod external_models;
mod specs;
