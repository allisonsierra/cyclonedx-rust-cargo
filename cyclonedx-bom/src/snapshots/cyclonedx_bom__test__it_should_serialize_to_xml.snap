---
source: cyclonedx-bom/src/lib.rs
expression: actual_xml

---
<?xml version="1.0" encoding="utf-8"?>
<Bom xmlns="ns:http://cyclonedx.org/schema/bom/1.3" version="1" serialNumber="fake-uuid" />
